sdk/python/kfp/v2/compiler_tests/test_data/two_step_pipeline_with_importer.json

```diff
@@ -0,0 +1,148 @@
+{
+  "pipelineInfo": {
+    "name": "two-step-pipeline-with-importer"
+  },
+  "tasks": [
+    {
+      "taskInfo": {
+        "name": "Trainer"
+      },
+      "inputs": {
+        "parameters": {
+          "num_epochs": {
+            "runtimeValue": {
+              "runtimeParameter": "epochs"
+            }
+          },
+          "train_optimizer": {
+            "runtimeValue": {
+              "runtimeParameter": "optimizer"
+            }
+          }
+        },
+        "artifacts": {
+          "input_location": {
+            "producerTask": "Trainer_input_location_importer",
+            "outputArtifactKey": "result"
+          }
+        }
+      },
+      "outputs": {
+        "parameters": {
+          "model_config": {
+            "type": "STRING"
+          }
+        },
+        "artifacts": {
+          "model_output": {
+            "artifactType": {
+              "instanceSchema": "title: Model\ntype: object\nproperties:\n"
+            }
+          }
+        }
+      },
+      "executorLabel": "Trainer"
+    },
+    {
+      "taskInfo": {
+        "name": "Serving"
+      },
+      "inputs": {
+        "parameters": {
+          "model_cfg": {
+            "runtimeValue": {
+              "runtimeParameter": "model_config"
```

Comment:
Why is this a runtime parameter? Read from below it should be connected to `Trainer`, right?

Reply:
Good catch, there's actually a bug. Fixed.